app/javascript/gobierto_dashboards/modules/contracts_controller.js

```diff
@@ -153,9 +152,20 @@ export class ContractsController {
       contract.start_date_year = contract.start_date ? (new Date(contract.start_date).getFullYear()) : contract.start_date;
     }
 
+    for(let i = 0; i < tendersData.length; i++){
+      const tender = tendersData[i];
+      const initial_amount = (tender.initial_amount === '' || tender.initial_amount === undefined) ? 0.0 : parseFloat(tender.initial_amount);
+
+      tender.initial_amount = initial_amount;
+      tender.submission_date_year = tender.submission_date != undefined && tender.submission_date != '' ? (new Date(tender.submission_date).getFullYear()) : tender.submission_date;
```

Comment:
initial_amount: three equals
submission_date_year: two equal

Btw, comparing again undefined and empty

You really do need a sublime plugin. Prettier, https://github.com/danreeves/sublime-prettier

Reply:
Fixed. Installed that too. Thanks!